src/Azs.Backup.Admin/test/Utils.ps1

```diff
@@ -11,8 +11,6 @@ function setupEnv() {
     # as default. You could change them if needed.
```

Comment:
Is this file used at all? I don't see any references to this file.

Reply:
It is used to set the env file, we will keep this file.